src/stlink-lib/flash_loader.c

```diff
@@ -3,213 +3,210 @@
 #include <unistd.h>
 
 #include <stlink.h>
+#include "flash_loader.h"
```

Comment:
We can leave the brackets < > here, as our makefile explicitly takes care of finding this header file.
Thus it is unified throughout the codebase:

Reply:
Correction: My mistake. It's ok like this. Just leave it as it is.